ariprog/ariprog.py

```diff
@@ -0,0 +1,41 @@
+"""
+ID: whuan2001
+LANG: PYTHON2
+TASK: ariprog
+"""
+def solution():
+    fin = open("ariprog.in", "r")
+    fout = open("ariprog.out", "w")
+    length, limit = map(int, fin.readlines())
+    squares = []
+    for l in range(limit + 1):
+        squares.append(l * l)
+    dic = set()
+    bisquares = []
+#    print "a"
```

Comment:
Sorry for mention that since I add these code, but please remove all these debugging comments